StantDrawing/BrushTool/BrushToolView.swift

```diff
@@ -66,15 +77,15 @@ public class BrushToolView: UIView {
     }
     
     func addAsSubViewOn(_ view: UIView, x: CGFloat, heightBaseToCenter: CGFloat) {
-        let y = (heightBaseToCenter - groupPencilSizeView.frame.height) / 2
-        let point = CGPoint(x: x, y: y - 60)
-        
         loadButtons()
         configureLayouts()
         if let uiButton = groupToolsButtons.first?.uiButton {
             setBtnAsClicked(button: uiButton)
         }
         
+        let y = (heightBaseToCenter - groupPencilSizeView.frame.height) / 2
+        let point = CGPoint(x: x, y: y)
```

Comment:
As you said before, this function (addAsSubViewOn) is also used in other context. Does this change have impacts elsewhere?

Reply:
Yes, but I refactored the code to remove those lines as we talked before.